research/backend/manager/dbmanager.py

```diff
@@ -0,0 +1,6 @@
+import redis
```

Comment:
Why don't you make this code to class representation? It'll be nice for managing and repairing source code.

Reply:
I expect you to receive the framework  :)
I'll apply it to mine !!